src/pyodide/internal/python.ts

```diff
@@ -110,6 +110,18 @@ export async function loadPyodide(
     (msg) => console.log(msg),
     (msg) => console.error(msg)
   );
+  pyodide.runPython(`
+    def _tmp():
+      import sys
+      from pathlib import Path
 
+      VENDOR_PATH = "/session/metadata/vendor"
+      print(VENDOR_PATH, Path(VENDOR_PATH).is_dir())
+      if Path(VENDOR_PATH).is_dir():
+        sys.path.append(VENDOR_PATH)
+
+    _tmp()
+    del _tmp
+  `);
```

Comment:
Nit: stick this into a separate function and then call it from loadPyodide (just to keep it cleaner)